test/vinyl/gh-5823-skip-newer-than-snap-vylog.result

```diff
@@ -7,9 +7,7 @@ test_run = require('test_run').new()
 -- 1. There's both memtx and vinyl data;
 -- 2. User starts checkpoint process;
 -- 3. In the most unsuitable moment instance crashes;
--- 4. Recovering in the casual mode does not help;
--- 5. Recovering in the force recovery mode solves the problem (deletes
---    redundant vylog file).
+-- 4. Recovering in the casual mode successes.
```

Comment:
What about having data after the snapshot record?

Reply:
Generating such vylog seems to be quite sophisticated for such insignificant (I mean rare - to get this bug user must be extremely unlucky) bug fix. We should either somehow edit last vylog in order to insert some valid rows or fix vclock in the reference (saved somewhere) vylog. Both options dont worth the effort IMHO. If you still believe that we should have this test case and/or can come up with easier way - let me know.

Reply:
Up to you, I won't argue.